src/components/Facets/filters.css

```diff
@@ -32,6 +32,14 @@
   content: '-';
 }
 
+.mobile-filters-container .range-input {
+  border: 1px solid rgba(0, 0, 0, 0.1)
+}
+
+.mobile-filters-container .price-range-submit {
+  font-family: var(--ff-gt-america-expanded)
```

Comment:
we should not reference fonts directly but through the variables here:  https://github.com/wilson-ecommerce/eds-amersports/blob/975ab0064771e46e16996b70318a67cb4200b9fc/styles/styles-wilson/styles.css#L27